x-pack/test/apm_api_integration/tests/transactions/transactions_groups_main_statistics.ts

```diff
@@ -98,18 +98,18 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         );
         expectSnapshot(impacts).toMatchInline(`
           Array [
-            98.5616469236242,
```

Comment:
How much work would it be to migrate to synthtrace and removing the snapshots first?

Reply:
based on the number of snapshots and files updated I'd say a lot.